libs/domain/picking/src/components/picking-header/picking-header.component.ts

```diff
@@ -0,0 +1,64 @@
+import { resolve } from '@spryker-oryx/di';
+import { RouterService } from '@spryker-oryx/router';
+import { IconTypes } from '@spryker-oryx/ui/icon';
+import { subscribe } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { state } from 'lit/decorators.js';
+import { tap } from 'rxjs';
+import { PickingListMixin } from '../../mixins';
+import { styles } from './picking-header.styles';
+
+export class PickingHeaderComponent extends PickingListMixin(LitElement) {
+  static styles = styles;
+
+  protected routerService = resolve(RouterService);
+
+  @state()
+  protected cartNote?: string;
+
+  @subscribe()
+  protected subscription$ = this.pickingList$.pipe(
+    tap((list) => {
+      this.cartNote = list.cartNote;
+    })
+  );
+
+  protected renderCartNoteButton(): TemplateResult {
+    return html`${this.cartNote
+      ? html` <oryx-icon-button
+          ><button aria-label="Show customer note">
+            <oryx-icon type="info"></oryx-icon></button
+        ></oryx-icon-button>`
+      : ''}`;
+  }
+
+  protected override render(): TemplateResult {
+    return html`<div class="title">
+        <button
```

Comment:
use oryx-button with type="text", please

Reply:
I want to, but it's the wrong color.

Reply:
Then better to align with Nuwan and update the designs. Because this is definitely an oryx-button type text and we have to avoid such custom solution as much as possible

Reply:
Resolved to use `oryx-icon-button` instead since the order ID is not meant to be interactive anyways.